app/konnect/getting-started/migration.md

```diff
@@ -0,0 +1,318 @@
+
+As organizations grow and scale they need for more advanced capabilities while lowering 
+operational complexity. Features such as strong [multi-tenancy](https://konghq.com/blog/enterprise/multi-tenancy), 
+federated API management, advanced security integrations, and others. With {{site.konnect_product_name}}, 
+users have access to a full featured API management platform which provides managed components and 
+builds on the lessons learned from deep customer usage of {{site.base_gateway}} and API management best practices. 
+
+This document will provide a guide for {{site.base_gateway}} users looking to migrate to {{site.konnect_product_name}}. 
+
+## Migration Guide
+
+The following will provide key details for completing a successful migration from {{site.base_gateway}} on-premises to 
+{{site.konnect_product_name}}. This document focuses on migrating a [Hybrid](/gateway/latest/production/deployment-topologies/hybrid-mode/) 
+or [Traditional](/gateway/latest/production/deployment-topologies/traditional/)
+deployment to Konnect, for other deployment modes see the following:
+
+* [{{site.kic_product_name}} (KIC)](/kubernetes-ingress-controller/latest/)
+    
+    If you run KIC on premises, migrating to Konnect is straightforward. 
+    The [Kong Ingress Controller for Kubernetes Assocation](/konnect/gateway-manager/kic/)
+    documentation provides details on linking your KIC deployment to a KIC based Control Plane
+    in {{site.konnect_product_name}}.
+
+* [DB-less mode](/gateway/latest/production/deployment-topologies/db-less-and-declarative-config/)
+
+    If you're running a DB-less deployment and interested in migrating to Konnect, Kong recommends migrating to 
+    Hybrid mode deployment to take advantage of the full capabilities of Kong Konnect. Please reach out to
+    a Kong representative for assistance with this migration.
+
+### Role Based Access Controls
+
+Both {{site.base_gateway}} and {{site.konnect_product_name}} provide frameworks for controlling security and
+data access. 
+
+{{site.base_gateway}} provides a traditional Role Based Access Control (RBAC) system
+to manage administrators and users of the API gateway system. {{site.konnect_product_name}} provides a robust 
+multi-layer Identity and Access Management (IAM) system including organizations, teams and roles. 
+Konnect also provides integrations with IdP providers via OIDC allowing you to map centrally managed teams to 
+Konnect based roles.
+
+{{site.base_gateway}}'s RBAC system does not map directly to the IAM system provided by {{site.konnect_product_name}}. 
+Users migrating from {{site.base_gateway}} on-premises to {{site.konnect_product_name}} have typically 
+chosen to use Konnect's IdP integrations and take advantage of their existing IdP solution and Konnect team 
+based mappings instead migrating their {{site.base_gateway}} on-premises RBAC configuration directly.
```

Comment:
```suggestion
When migrating from a self-managed {{site.base_gateway}} to {{site.konnect_short_name}}, 
we recommend using {{site.konnect_short_name}}'s IdP integrations. 
You can take advantage of your existing IdP solution and {{site.konnect_short_name}}'s 
team-based mappings instead of migrating your self-managed {{site.base_gateway}} 
RBAC configuration directly.
```